test/e2e/utils.go

```diff
@@ -386,6 +387,12 @@ func getProviders() ([]TestProvider, error) {
 						},
 					},
 				}
+				fakegcsenabled := getEnvOrFallback("GOOGLE_EMULATOR_ENABLED", "")
```

Comment:
The defined constant for the environment variable can be used here.

Reply:
Addressed in latest commit [`585d628`](https://github.com/gardener/etcd-druid/pull/741/commits/585d6286183f22ee5700e9bef3b08c2166a3f4ba)